CHANGELOG.md

```diff
@@ -1,3 +1,200 @@
+  * Revert "Revert "fix(webview): authenticationMethod NSURLAuthenticationMethodClientCertificate not handled correctly (#13352)" (#13354)" (#13367) ([2e8fa00](https://github.com/tidev/titanium_mobile/commit/2e8fa00f24f4c18e766f10eaee3d9506b3ac57a3))
+  * Revert "fix(webview): authenticationMethod NSURLAuthenticationMethodClientCertificate not handled correctly (#13352)" (#13354) ([827752f](https://github.com/tidev/titanium_mobile/commit/827752f46a25209c58e167e69a37947040ec0c6b))
+  * support flexible border radius in card view ([a1c8edf](https://github.com/tidev/titanium_mobile/commit/a1c8edf55938186c8b9e500e72e264533a1e0974))
+  * fix build deps ([dbb7177](https://github.com/tidev/titanium_mobile/commit/dbb7177498d434983eee0d720cac8a31238f6632))
+  * redraw CGColor when changing trait collection ([1ac97a4](https://github.com/tidev/titanium_mobile/commit/1ac97a45283e2ecdd87d680a70b9d5e153e38c0b))
+
+* Marc Bender
+  * fixes crashing macos app because of permissions ([c36867e](https://github.com/tidev/titanium_mobile/commit/c36867e6422246119a7f3a76566771fe52bfa6db))
+  * textarea contentHeight fix with new return property textareaHeight in "change" event ([94820d1](https://github.com/tidev/titanium_mobile/commit/94820d1952d5e301b6ea2741576b3046b5b382b5))
+  * backgroundcolor Ti.UI.backgroundcolor fixed on modal windows ([a1c83d8](https://github.com/tidev/titanium_mobile/commit/a1c83d8e0a5b071b71ede22d494447949656f639))
+
+* chauf
+  * authenticationMethod NSURLAuthenticationMethodClientCertificate not handled correctly ([edfd037](https://github.com/tidev/titanium_mobile/commit/edfd037c4e6f50aa03cd382247d5128bb9292731))
+
+* Sergey Volkov
+  * update link to a build status badge ([b66c287](https://github.com/tidev/titanium_mobile/commit/b66c287161788e00d05adbe5bc2b61322d827688))
+  * problems with "userinterfacestyle" event ([bdfd640](https://github.com/tidev/titanium_mobile/commit/bdfd64029484abd2f3b61b1239c6cd3835e82722))
+  * [TIMOB-28563](https://jira-archive.titaniumsdk.com/TIMOB-28563) - use Activity for colors resolution ([e290733](https://github.com/tidev/titanium_mobile/commit/e2907331ef681ca2aaf6d96d1e1718dbeb427acc))
+  * reuse address for debugger ([c404163](https://github.com/tidev/titanium_mobile/commit/c4041636297034167bc581474c7057acf5ebae7f))
+  * [TIMOB-28583](https://jira-archive.titaniumsdk.com/TIMOB-28583) - commit fragments transaction only once after wrapping view being attached to a window ([0838db2](https://github.com/tidev/titanium_mobile/commit/0838db24a64d0eec534d02b9b43efea8d98d7d05))
+  * crash in ListView on API 21 ([b30de10](https://github.com/tidev/titanium_mobile/commit/b30de101032fc668bcf8284559aec3f0a4eacb52))
+  * Titanium.UI.iOS.Toolbar was removed in 10.0.0 ([23986a3](https://github.com/tidev/titanium_mobile/commit/23986a36be11b52a8485ce8971e6e3869c7bc738))
+
+* m1ga
+  * update documentation links in readme ([79a94f7](https://github.com/tidev/titanium_mobile/commit/79a94f746b4971af4b10416761b779c8a431ed04))
```

Comment:
can you move that into my other section above? Not sure why this is separate.

Reply:
I already did in my next PR for the official release note in the docs, but I can also do it here

Reply:
ah no problem!
just wanted to add that you'll need to look at https://github.com/tidev/titanium-docs/tree/main/docs/guide/Titanium_SDK/Titanium_SDK_Release_Notes/Titanium_SDK_Release_Notes_10.x too :smile: 
And keep in mind: there will be a RC before the GA so it won't be the official release